xblock/utils/resources.py

```diff
@@ -57,7 +56,8 @@ def render_mako_template(self, template_path, context=None):
         )
         context = context or {}
         template_str = self.load_unicode(template_path)
-        lookup = MakoTemplateLookup(directories=[pkg_resources.resource_filename(self.module_name, '')])
+        directory = os.path.dirname(os.path.realpath(sys.modules[self.module_name].__file__))
```

Comment:
Since `sys.modules` only includes modules that have already been imported, this will fail if `self.module_name` hasn't been imported yet, which is unlikely, but possible. So, I would use `importlib.import_module` just like you did in `load_unicode`, and then grab `__file__` from the result of that.